gluonnlp/data/sentiment.py

```diff
@@ -69,6 +70,203 @@ def _get_data(self):
             download(_get_repo_file_url('gluon/dataset/imdb', data_file_name),
                      path=root, sha1_hash=data_hash)
 
+    def _read_data(self):
+        with open(os.path.join(self._root, self._segment + '.json')) as f:
+            samples = json.load(f)
+        return samples
+
+class MR(SimpleDataset):
+    """Movie reviews for sentiment analysis.
+
+    From
+    https://www.cs.cornell.edu/people/pabo/movie-review-data/
+
+    Parameters
+    ----------
+    segment : str, default 'train'
+        Dataset segment. Options are 'train' and 'test'.
+    root : str, default '$MXNET_HOME/datasets/mr'
+        Path to temp folder for storing data.
+        MXNET_HOME defaults to '~/.mxnet'.
+    """
+    def __init__(self, segment='all', root=os.path.join(_get_home_dir(), 'datasets', 'mr')):
```

Comment:
seems that only "all" segment is supported. in that case, the segment argument can be removed.